packages/content-conformance/package.json

```diff
@@ -16,9 +16,14 @@
     "find-up": "^6.3.0",
     "globby": "^13.1.2",
     "remark-mdx": "^1.6.22",
+    "ts-node": "^10.9.1",
```

Comment:
Was this for testing purposes? If so, we might be able to use our `hc-tools` CLI (a light wrapper around `ts-node`) to run this without building.

Reply:
Removed!